<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWidget</class>
 <widget class="QWidget" name="settingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="settings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1031</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#settings {
	background-color: #2a2a2a;
	border-radius: 15px;
}


#leftArrow,
#rightArrow {
  border-radius: 2px;
  color: #DCE4EE;
  background: none;
}

#leftArrow:pressed,
#rightArrow:pressed {
  color: #959aa1;
  background: rgba(0, 0, 0, 0.5);
}


QGroupBox {
	border: 2px solid #DCE4EE;
    border-radius: 5px;
	margin-top: 0.5em;
}
QGroupBox::title {
	top: -10px;
	left: 10px;
	padding: 0 2px ;
	color: #DCE4EE;
}


/* Label */
QLabel {
	color: #DCE4EE;
}


/* PushButton */
QPushButton {
	color: #DCE4EE;
	background-color: #2f5ea1;
	border-radius: 8px;
}
QPushButton:pressed {
	color: #DCE4EE;
	background-color: #20406d;
}
QPushButton:disabled {
	color: #959aa1;
	background-color: #183154;
}
#stopButton {
	background-color: #9e2c24;
}
#stopButton:pressed {
	background-color: #6b1d18;
}


/* LineEdit */
QLineEdit {
	color: #DCE4EE;
	background-color: rgba(255, 255, 255, 0.1);
	border-radius: 3px;
	border: 1px solid #DCE4EE;
}
QLineEdit:disabled {
	color: #959aa1;
	background-color: rgba(255, 255, 255, 0.05);
	border: 1px solid #959aa1;
}


/* ComboBox */
QComboBox {
	border-radius: 0px;
	color: #DCE4EE;
	background-color: rgba(255, 255, 255, 0.1);
}
QComboBox:disabled {
	color: #959aa1;
	background-color: rgba(255, 255, 255, 0.05);
}
QComboBox QListView {
	padding: 5px;
	color: #DCE4EE;
	background-color: rgba(255, 255, 255, 0.1);
	border-radius: 0px;
	outline: 0px;
}
QComboBox QListView:disabled {
	color: #959aa1;
	background-color: rgba(255, 255, 255, 0.05);
}


/* CheckBox */
QCheckBox {
	color: #DCE4EE;
}
QCheckBox:disabled {
	color: #959aa1;
}


/* PlainTextEdit */
QPlainTextEdit {
	padding: 3px;
	border: 1px solid #DCE4EE;
	border-radius: 10px;
	color: #DCE4EE;
	background-color: rgba(255, 255, 255, 0.1);
}
QPlainTextEdit:disabled {
	border: 1px solid #959aa1;
	color: #959aa1;
	background-color: rgba(255, 255, 255, 0.05);
}
/* SCROLLBAR */
QScrollBar {
	border: none;
    background: #3f3f3f;
	border-radius: 0px;
}
QScrollBar:vertical {
    width: 10px;
 }
QScrollBar:horizontal {
    height: 10px;
}
QScrollBar:disabled {
    background: rgba(255, 255, 255, 0.0);
 }
/* HANDLE BAR */
QScrollBar::handle {
	background: #DCE4EE;
	border-radius: 5px;
}
QScrollBar::handle:pressed {	
	background-color: #acb3bb;
}
QScrollBar::handle:disabled {
	background: #959aa1;
}
/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical, 
QScrollBar::add-line:vertical {
	height: 0px;
}
/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:horizontal, 
QScrollBar::add-line:horizontal {
	width: 0px;
}
/* RESET ARROW */
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical,
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical,
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal,
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	background: none;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="miscSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Misc settings</string>
    </property>
    <widget class="QPushButton" name="modListImportButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Import mod list</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modListExportButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Export mod list</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="apiSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>951</width>
      <height>61</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="apiKeyApplyButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QLabel" name="apiKeyLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-1</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CurseForge API Key</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="apiKeyInput">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>781</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Put your CurseForge API key here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apiKeyHelpButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="apiSettingsArrows">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="rightArrow">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Next API</string>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftArrow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Previous API</string>
     </property>
     <property name="text">
      <string>◀</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
